lib/auth0/api/v2/jobs.rb

```diff
@@ -5,16 +5,28 @@ module V2
       module Jobs
         # https://auth0.com/docs/apiv2#!/jobs/get_jobs_by_job_id
         def get_job(job_id)
+          fail Auth0::InvalidParameter, 'you must specify a job id' if job_id.to_s.empty?
           path = "/api/v2/jobs/#{job_id}"
```

Comment:
Does it makes sense to have an API default url to avoid duplication on /api/v2/ on every path?